### DISCUSSION TITLE
DevTools for GridEngine

### DISCUSSION THREAD

**Discussion by zewa666** [Annoraaq/grid-engine#261] [Ideas]
Hey everyone,

I've been now quite often in the situation where it would be rather cool to have a quick means to update the current games state while debugging. So I thought, hey why not add something to the DevTools themselves?

So I've just quickly built a small demo to show what I'm thinking off.

https://user-images.githubusercontent.com/2861414/161254248-ee7b2661-ca35-4d60-8e06-171fd053e3f9.mp4

Would you guys see something like this as a helpful addition to GridEngine?

**Comment by Annoraaq** [comment]
I think it is really cool! Very helpful for debugging and developing.

**Comment by zewa666** [comment]
Here's some progress.

https://user-images.githubusercontent.com/2861414/161390076-0611e648-5a30-4e65-96de-339b873fb987.mp4

Next steps, Chrome Store upload ;)

**Reply by Annoraaq** [reply]
Great work! It's already fun watching the video xD

**Reply by zewa666** [reply]
Hehe, good to hear. I've finally found all the steps required, and paid the 5 bucks to Google for the Dev-Account. Now lets see how long it takes 'em to check and proceed with publishing the Extension. My last extension is already a couple of years ago so no idea what it's going to look like nowadays ;)

**Comment by zewa666** [comment]
finally its done. you can find the extension for Chrome over here https://chrome.google.com/webstore/detail/moplhbafnakhjjehlmpedbmbomcmgope

additionally, the related github repo here https://github.com/zewa666/grid-engine-devtools

**Reply by Annoraaq** [reply]
Great to hear that. I will place a link in the readme so people know about it.
